pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -632,13 +632,36 @@ func (og *operationGenerator) resizeFileSystem(volumeToMount VolumeToMount, devi
 				return nil, nil
 			}
 
-			diskFormatter := &mount.SafeFormatAndMount{
-				Interface: mounter,
-				Exec:      og.volumePluginMgr.Host.GetExec(expandableVolumePlugin.GetPluginName()),
+			var resizeStatus bool
+			var resizeErr error
+
+			fsResizablePlugin, callExpandFS := expandableVolumePlugin.(volume.FSResizableVolumePlugin)
```

Comment:
Didn't we discuss that `ExpandFS` call should be implemented by all plugins that want to expand file system? In most cases, the call with do nothing other than call out to resize util function but it will enable more polymorphic behaviour.

Reply:
@gnufied agree and we had two options 

1. Add `ExpandFS` to all in-tree plugins right away and then add boilerplate code in there which calls `resizefs.NewResizer()` etc 
2. Do a lazy approach where if a plugin hasn't implemented that method, `resizefs.NewResizer()...` gets called at the operationGenerator level by default, and if a plugin _does_ implement `ExpandFS` method then it automatically will get called - isn't that polymorphic.

I can go with #1 if you want me to, but don't you think we are being polymorphic as intended?

Reply:
+1 for @gnufied. Each plugin should have this call.

Reply:
@gnufied @chakri-nelluri OK that's fine. I will change it and go with #1